specification/indices/get_sample/GetRandomSampleRequest.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RequestBase } from '@_types/Base'
+import { IndexName } from '@_types/common'
+
+/**
+ * Request for a random sample of raw documents ingested into the given index or data stream.
+ *
+ * @rest_spec_name indices.get_sample
+ * @availability serverless visibility=feature_flag feature_flag=random_sampling since=9.3.0 stability=experimental
```

Comment:
Just to clarify: this `@availability` annotation will make this API available in serverless _only_. It will show up in the clients as not supported for stack unless you also add a stack `@availability` annotation.